pages/ecosystem.vue

```diff
@@ -141,104 +160,156 @@ useSeoMeta({
   ogUrl: `${config.public.siteUrl}/ecosystem/`,
 });
 
-const tierFilters = ref<string[]>([]);
+const categoryFilters = ref<string[]>([]);
+const selectedTab = ref<string>("Main");
+const searchedText = ref<string>("");
+const propToSortBy = ref<string>("name");
 
-const tierFiltersAsString = computed(() => tierFilters.value.join(","));
+const tiersNames = tiers.map((tier) => tier.name);
+const membersCategories = members.map((member) => member.labels);
+const sortedMembersCategories = [...new Set(membersCategories.flat())].sort(
+  (a, b) => a.localeCompare(b)
+);
+const membersByTier: MembersByTier = tiersNames.reduce((acc, tierName) => {
+  return {
+    ...acc,
+    ...{ [tierName]: members.filter((member) => member.tier === tierName) },
+  };
+}, {});
 
-const filteredMembers = computed(() => {
+const categoryFiltersAsString = computed(() => categoryFilters.value.join(","));
+
+function getSelectedTierDescription() {
+  const tier = tiers.find((tier) => tier.name === selectedTab.value);
+  return tier?.description || "";
+}
+
+function getFilteredMembers(tierName: string) {
   if (!members) {
     return [];
   }
 
-  const noTierFilters = tierFilters.value.length === 0;
+  const filteredMembersByTier = membersByTier[tierName];
 
-  return noTierFilters
-    ? members
-    : members.filter((member) => tierFilters.value.includes(member.tier));
-});
+  let result = filteredMembersByTier;
 
-const tiersNames = computed(() => tiers.map((tier) => tier.name));
+  if (searchedText.value !== "") {
+    result = result.filter((member) =>
+      member.description
+        .toLowerCase()
+        .includes(searchedText.value.toLowerCase())
+    );
+  }
+
+  if (categoryFilters.value.length > 0) {
+    result = result.filter((member) =>
+      categoryFilters.value.every((filter) => member.labels.includes(filter))
```

Comment:
```suggestion
      categoryFilters.value.some((filter) => member.labels.includes(filter))
```

this will update the function so that any member that has any of the checked filters returns in the results